docker/sandbox/Dockerfile

```diff
@@ -56,37 +56,6 @@ COPY charts/flyte-core/ /flyteorg/share/flyte-core
 # Copy scripts
 COPY docker/sandbox/kubectl docker/sandbox/cgroup-v2-hack.sh docker/sandbox/wait-for-flyte.sh /flyteorg/bin/
 
-
-FROM base_ AS default
```

Comment:
was this just not used?  I don't see a `--from=default` anywhere.  Or do you not need to specify that because it's default?

Reply:
yeah we can simplify it